main.go

```diff
@@ -22,9 +22,41 @@ func main() {
 	window.Set("reverse", js.FuncOf(reverse))
 	window.Set("filter", js.FuncOf(filter))
 	window.Set("extension", js.FuncOf(extension))
+	window.Set("tile", js.FuncOf(tile))
 	<-ch
 }
 
+func tile(_ js.Value, _ []js.Value) interface{} {
+	f := func(bc imgedit.ByteConverter) error {
+		xLengthInput := getElementById("x-length")
+		yLengthInput := getElementById("y-length")
+
+		xLength, err := strconv.Atoi(xLengthInput.Get("value").String())
+		if err != nil {
+			return nil
+		}
+		yLength, err := strconv.Atoi(yLengthInput.Get("value").String())
+		if err != nil {
+			return nil
+		}
+		xLengthMax, err := strconv.Atoi(xLengthInput.Get("max").String())
+		if err != nil {
+			return nil
+		}
+		yLengthMax, err := strconv.Atoi(yLengthInput.Get("max").String())
+		if err != nil {
+			return nil
+		}
```

Comment:
This is a very clever way of synchronizing with the browser side.

It might be a good idea to incorporate this approach in resize and trim as well.
On the other hand, it seems to be based on the assumption that the browser's values are to be trusted.

This page is based on a **wasm**, so we are not bothered by invalid values (it only consumes unnecessary resources on the user side).

Would you tell me your opinion on this matter if you don't mind?

Reply:
> On the other hand, it seems to be based on the assumption that the browser's values are to be trusted.
You are certainly right.

I think a better implementation would be to leave the max attribute of Input set, and when the process is executed, the go side also gets the image size and calculates the max value again. 
...However, I don't know how to get the image size on the go side.(Also, it's a bit of a pain :no_mouth:)

> This page is based on a wasm, so we are not bothered by invalid values (it only consumes unnecessary resources on the user side).

I think this is also true, so I think it is fine to keep the current implementation (trusting the value sent by the browser).